go/cmd/betabot/main.go

```diff
@@ -57,64 +62,326 @@ func betabot() error {
+					})
+					if err != nil {
+						log.Printf("handle event: %v", err)
+						return
+					}
+
+					time.Sleep(1 * time.Second)
+					userMessage, err := proto.Marshal(&bertymessenger.AppMessage_UserMessage{
+						Body: "OK, it’s done! Welcome here, and congrats for joining our community! 👏👍🔥\nType /help when you need infos about available test commands! 📖",
+					})
+					if err != nil {
+						log.Printf("handle event: %v", err)
+						return
+					}
+					_, err = messengerClient.Interact(ctx, &bertymessenger.Interact_Request{
+						Type:                  bertymessenger.AppMessage_TypeUserMessage,
+						Payload:               userMessage,
+						ConversationPublicKey: interaction.ConversationPublicKey,
+					})
+					if err != nil {
+						log.Printf("handle event: %v", err)
+						return
+					}
+
+					// delete the conv in convs
+					convs = append(convs[:idx], convs[idx+1:]...)
+					log.Printf("Finished !%v", convs)
+					return
+				}
+			}
+
+			answers := []string{
+				"Welcome to the beta!",
+				"Hello! Welcome to Berty!",
+				"Hey, I hope you're feeling well here!",
+				"Hi, I'm here for you at anytime for tests!",
+				"Hello dude!",
+				"Hello :)",
+				"Ow, I like to receive test messages <3",
+				"What's up ?",
+				"How r u ?",
+				"Hello, 1-2, 1-2, check, check?!",
+				"Do you copy ?",
+				"If you say ping, I'll say pong.",
+				"I'm faster than you at sending message :)",
+				"One day, bots will rules the world. Or not.",
+				"You're so cute.",
+				"I like discuss with you, I feel more and more clever.",
+				"I'm so happy to chat with you.",
+				"I could chat with you all day long.",
+				"Yes darling ? Can I help you ?",
+				"OK, copy that.",
+				"OK, I understand.",
+				"Hmmm, Hmmmm. One more time ?",
+				"I think you're the most clever human I know.",
+				"I missed you babe.",
+				"OK, don't send me nudes, I'm a bot dude.",
+				"Come on, let's party.",
+				"May we have a chat about our love relationship future ?",
+				"That's cool. I copy.",
+			} // 28
+
+			if receivedMessage.GetBody() == "/help" {
+				userMessage, err := proto.Marshal(&bertymessenger.AppMessage_UserMessage{
+					Body: "In this conversation, you can type all theses commands :\n/demo group\n/demo demo\n/demo share\n/demo contact \"Here is the QR code of manfred, just add him!\"",
+				})
+				if err != nil {
+					log.Printf("handle event: %v", err)
+					return
+				}
+
+				_, err = messengerClient.Interact(ctx, &bertymessenger.Interact_Request{
+					Type:                  bertymessenger.AppMessage_TypeUserMessage,
+					Payload:               userMessage,
+					ConversationPublicKey: interaction.ConversationPublicKey,
+				})
+				if err != nil {
+					log.Printf("handle event: %v", err)
+					return
+				}
+				return
+			}
+			// auto-reply to user's messages
+			nBig, err := rand.Int(rand.Reader, big.NewInt(28))
+			if err != nil {
+				log.Printf("Failed to generate randome number: %v", err)
+				return
+			}
```

Comment:
I suggest you to init the seed with `rand.Seed(srand.MustSecure())` at the top of the betabot() function

then you can just use `answers[rand.Intn(len(answers))]`